tbx/static_src/sass/components/_stats.scss

```diff
@@ -32,10 +36,10 @@
 
     &__number {
         @include font-size(stat);
-        font-weight: $weight--light;
         color: var(--color--heading);
         display: block;
-        margin-bottom: $spacer-mini;
+        font-weight: $weight--light;
```

Comment:
Nitpick - I notice you re-ordered the properties to be alphabetical order. I tend to roughly group mine by type - as the font-size include has to come first I then put the font-weight after it. It's not an exact science. However, this is only to explain my approach I don't mind the reordering :)